concent_api/core/payments/backends/sci_backend.py

```diff
@@ -80,16 +80,22 @@ def make_force_payment_to_provider(
 
 
 def is_account_status_positive(
```

Comment:
quite frankly, it should be renamed to:
`has_client_enough_funds` or something similar

Reply:
:+1: done